crawler/plugins/applications/tomcat/tomcat_container_crawler.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+import dockercontainer
+from icrawl_plugin import IContainerCrawler
+from plugins.applications.tomcat import tomcat_crawler
+from utils.crawler_exceptions import CrawlError
+
+logger = logging.getLogger('crawlutils')
+
+
+class TomcatContainerCrawler(IContainerCrawler):
+    feature_type = 'application'
+    feature_key = 'tomcat'
+    default_port = 8080
+
+    def get_feature(self):
+        return self.feature_key
+
+    def crawl(self, container_id=None, **kwargs):
+        password = "password"
+        user = "tomcat"
+
+        if "password" in kwargs:
+            password = kwargs["password"]
+
+        if "user" in kwargs:
+            user = kwargs["user"]
+
+        c = dockercontainer.DockerContainer(container_id)
+
+        # check image name
+        if c.image_name.find(self.feature_key) == -1:
```

Comment:
Why is this check really needed? Even if app name is not encoded in image name, if there is proper condition testing in the metrics extraction code, wouldn't the frame just be empty?